internal/slicer/slicer_test.go

```diff
@@ -616,6 +616,19 @@ var slicerTests = []slicerTest{{
 }, {
 	summary: "Can list parent directories of normal paths",
 	slices:  []setup.SliceKey{{"test-package", "myslice"}},
+	pkgs: map[string][]byte{
+		"test-package": testutil.MustMakeDeb(append(
+			testutil.TestPackageEntries,
+			// This particular path starting with "/check" is chosen to test for
+			// a particular bug; which appeared due to the usage of
+			// strings.TrimLeft() instead strings.TrimPrefix() to determine a
+			// relative path. Since TrimLeft takes in a cutset instead of a
+			// prefix, the desired relative path was not produced. The check.v1
+			// module creates temporary path with the word "check" within. Thus,
+			// this particular path reproduced the bug.
+			testutil.Dir(0755, "./check-foo/"),
```

Comment:
Can we make this a separate regression test with only this entry? That might be easier to read and the output of the tests would be much more self-contained.

Reply:
Moved in https://github.com/canonical/chisel/pull/145/commits/5fe11069c55975da90c79d69b1dd705389302e49.